consensus/polybft/mocks_test.go

```diff
@@ -284,15 +284,15 @@ func (s *stateProviderMock) Txn(ethgo.Address, ethgo.Key, []byte) (contract.Txn,
 	return nil, nil
 }
 
-var _ Transport = &transportMock{}
+// var _ Transport = &transportMock{}
 
-type transportMock struct {
-	mock.Mock
-}
+// type transportMock struct {
```

Comment:
do we need all this commented out code?

Reply:
Fixed [c14c710](https://github.com/0xPolygon/polygon-edge/pull/843/commits/c14c7100feaa8b6d220905a0b02c03632ca046ca)